src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -29,20 +29,11 @@ import { LocationType, FacilityType, BOUNDING_RADIUS } from '../constants';
 import { areGeocodeEqual /* areBoundsEqual */ } from '../utils/helpers';
 import { facilityLocatorShowCommunityCares } from '../utils/selectors';
 import { isProduction } from 'platform/site-wide/feature-toggles/selectors';
-import environments from '../../../platform/utilities/environment';
-/**
- New: mbxClient is the new instance for the API calls and
- The new SDK requires to pass the mapbox client to geocode services
- otherwise use the client as it is
- */
-let mbxClient;
-
-if (environments.isStaging()) {
-  const mbxGeo = require('@mapbox/mapbox-sdk/services/geocoding');
-  mbxClient = mbxGeo(mapboxClient);
-} else {
-  mbxClient = mapboxClient;
-}
+
+const mbxGeo = require('@mapbox/mapbox-sdk/services/geocoding');
```

Comment:
Any reason we can't do `import mbxGeo from '@mapbox/mapbox-sdk/services/geocoding';` here? 🙂

Reply:
I agree with Kelson.  This is JS for the browser, not Node, so ES6 imports should be used instead of CommonJS modules.